include/seastar/core/reactor_config.hh

```diff
@@ -33,6 +33,7 @@ struct reactor_config {
     bool auto_handle_sigint_sigterm = true;
     unsigned max_networking_aio_io_control_blocks = 10000;
     bool kernel_page_cache = false;
+    bool have_aio_fsync = false;
```

Comment:
This isn't reactor configuration because the calculation includes kernel_supports_aio_fsync().

The configuration is whether we want to use aio fsync if it's available or not. have_aio_fsync is the result of aio_fsync && kernel_supports, this isn't configuration the user can provide.

Reply:
Though I see reactor_config is marked internal. But is it really?

Reply:
Ah, maybe it's okay. We have app_template::config from which reactor_config is derived.

Reply:
> Though I see reactor_config is marked internal. But is it really?

i think it is, unless a seastar application creates an instance of `reactor` manually, instead of using `app_template`.

Reply:
It's not possible to create reactors directly.  The entry point is smp::configure but it's too difficult to use.